lib/scripts/createMeal.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:fitness_freaks/models/postTypes.dart';
```

Comment:
The script's looking good.  Love the way you put in a futuristic thought to it by adding youtube link etc.